t04_refactored.py

```diff
@@ -231,9 +231,36 @@ def team_19_adv():
 ###################################################################################
 
 def team_20_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1aO8daq4WLqOsChlhvmClBHNZEx4faJcmsvGXPyOU990/edit?usp=sharing
+    :return:
+    America Gaona Borges
+    Jessica Douthit
+    """
+    global isDead
+    input("Go with him further Up, or Down? ")
```

Comment:
The `input` isn't saved to a variable:

`direction = input("Go with him further Up, or Down? ")`